live-examples/js-examples/promise/promise-any.js

```diff
@@ -0,0 +1,9 @@
+const promise1 = Promise.reject(0);
+const promise2 = new Promise((resolve) => setTimeout(resolve, 100, 'quick'));
+const promise3 = new Promise((resolve) => setTimeout(resolve, 500, 'slow'));
+const promises = [promise1, promise2, promise3];
+
+Promise.any(promises).then((value) => console.log(value));
+
+// expected output:
+// "quick"
```

Comment:
I think this should be on one line, like:

```
// expected output: "quick"
```

...is there a reason I'm missing why you are not doing this here?

Reply:
No reason, just a result of moving things around in the process and not cleaning up afterwards. :)